plugins/rest/rest_handlers.go

```diff
@@ -504,16 +408,19 @@ func (plugin *Plugin) telemetryNodeCountHandler(formatter *render.Render) http.H
 	}
 }
 
-// indexHandler - used to get index page
-func (plugin *Plugin) indexHandler(formatter *render.Render) http.HandlerFunc {
+func (plugin *Plugin) registerIndexHandler(key, method string, indexes *indexes) {
```

Comment:
Not much useful having to define it like this, we should try to get rid of code where defining extra function around it doesn't save much. In `registerHTTPHandler` it might be useful since we use the dump functions inside, but here it's just unnecessary indirection since the data is already field in plugin.